internal/network/machine_network_cidr.go

```diff
@@ -117,7 +153,7 @@ func VerifyVips(hosts []*models.Host, machineNetworkCidr string, apiVip string,
 		err = VerifyVip(hosts, machineNetworkCidr, ingressVip, "ingress-vip", mustExist, log)
 	}
 	if err == nil {
-		err = VerifyDifferentVipAddresses(apiVip, ingressVip)
+		err = ValidateNoVIPAddressesDuplicates([]*models.APIVip{{IP: models.IP(apiVip)}}, []*models.IngressVip{{IP: models.IP(ingressVip)}})
```

Comment:
I'm curious why your IDE didn't show you this, but you can do a simplifications around your code. Instead
``
[]*models.APIVip{{IP: models.IP(apiVip)}}, []*models.IngressVip{{IP: models.IP(ingressVip)}}
```
you can write
```
[]*models.APIVip{{IP: apiVip}}, []*models.IngressVip{{IP: ingressVip}}
```
because `models.IP` is just a syntax sugar over `string`